v2/cli/init_templates.go

```diff
@@ -30,17 +30,24 @@ spec:
       brigade.{{ .Language }}: |
         {{- nindent 8 .Script }}
     {{- else }}
+    ## Your brigade.{{ .Language }} script will be loaded from the 
+    ## following git repository by default, but note that individual 
+    ## events CAN override these details. The Brigade GitHub Gateway, 
+    ## for instance, overrides these details for relevant events so the 
+    ## events themselves reference applicable pull branches, tags, pull
```

Comment:
Tiny mistake in this line (which probably came from me). Remove "pull" from "pull branches".